src/main/webapp/index.html

```diff
@@ -4,6 +4,6 @@
 <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
 <link rel="stylesheet" href="style.css">
 <script src="script.js"></script>
-<h1 class="display-1">The Open Source Community is for <span class="emphasis">Everyone</span></h1>
+<h1 class="display-1"><div id="home_header">The Open Source Community is for <span class="emphasis">Everyone</span></div></h1>
```

Comment:
maybe adding the ID to the h1 tag.
```suggestion
<h1 id= "home_header" class="display-1">The Open Source Community is for <span class="emphasis">Everyone</span></h1>
```

Reply:
Wow I had no clue you could have an id and a class on the same <>!!! Thanks

Reply:
no problem!